index.html

```diff
@@ -126,7 +127,35 @@
 				</p>
 			</div> <!-- cd-faq-content -->
 		</li>
-	</ul> <!-- cd-faq-group -->
+	</ul> 
+        <ul id="frameworki" class="cd-faq-group">
+                <li class="cd-faq-title"><h2>Frameworki</h2></li>
+                <li>
+                        <a class="cd-faq-trigger" href="#0">Laravel</a>
+                        <div class="cd-faq-content">
+                                <p>Eloquent (dostęp do bazy danych):</p>
+                                <ol>
+                                        <li>Dostępne są dwa query buildery: użycie <a href="https://github.com/laravel/framework/blob/v5.0.32/src/Illuminate/Database/Eloquent/Builder.php">pierwszego z nich</a> powoduje zwrócenie obiektów Active Record, natomiast <a href="https://github.com/laravel/framework/blob/v5.0.32/src/Illuminate/Database/Query/Builder.php">drugi</a> zwraca tablice.</li>
```

Comment:
Nie pełni roli repozytorium. Właśnie problem w tym, że nie wiadomo jaką "to coś" rolę pełni albo jaką próbuje pełnić. Wygląda mi to na coś w rodzaju Data Access Object, który wewnętrznie korzysta z QueryBuilder, który z kolei nie jest nawet builderem, po prostu implementuje fluent interface co jeszcze nie robi z niego buildera. Swaglor widzę nie tylko Fasady od Proxy nie odróżnia ale też myśli, że jak coś zwraca w każdej metodzie $this to jest builderem.